src/main/java/com/puppycrawl/tools/checkstyle/checks/blocks/NeedBracesCheck.java

```diff
@@ -66,6 +66,32 @@
  * <pre>
  * &lt;module name="NeedBraces"/&gt;
  * </pre>
+ * <p>Valid code example:</p>
+ * <pre>
+ * if (true) { // OK
+ *    return true;
+ * }  else { //OK
+ *    return false;
+ * }
+ * for(int i =0; ; ) { // OK
+ *    ++count;
+ * }
+ * do { // OK
+ *    ++count;
+ * } while (false);
+ * </pre>
+ * <p>Invalid code example:</p>
```

Comment:
No need for separate section, add OK and not OK examples in one section

Reply:
done.